packages/spotlight/SpotlightRootDecorator/SpotlightRootDecorator.js

```diff
@@ -104,9 +104,15 @@ const SpotlightRootDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			}
 		};
 
-		handleKeyDown = () => {
-			this.rootContainer.classList.add('mouse-mode');
-			this.rootContainer.classList.remove('touch-mode');
+		handleKeyDown = (ev) => {
+			const {keyCode} = ev;
+
+			if (keyCode === 13 && this.rootContainer.classList.contains('touch-mode')) {
```

Comment:
```suggestion
			if (is('enter', keyCode) && this.rootContainer.classList.contains('touch-mode')) {
```
We need to handle 13 and 16777221 for an _enter_ key.

Reply:
I applied your proposal to my PR. Thanks.